test/integration/multiplexed_integration_test.cc

```diff
@@ -136,13 +139,24 @@ TEST_P(Http2IntegrationTest, CodecStreamIdleTimeout) {
   envoy::config::core::v3::Http2ProtocolOptions http2_options =
       ::Envoy::Http2::Utility::initializeAndValidateOptions(
           envoy::config::core::v3::Http2ProtocolOptions());
-  http2_options.mutable_initial_stream_window_size()->set_value(65535);
+  http2_options.mutable_initial_stream_window_size()->set_value(32 * 1024);
```

Comment:
I think for H/2 the 64K is the minimum. The 32K will be ignored in this case and 64K is applied.

Reply:
Good point! In fact, the test still pass with H2. After looking to the log, it seems that before 64k flow control limit kicks in, socket already couldn't send all the data after sending ~32kB. I guess it passes because H2 hit congestion control window with a 32k payload. This is about the same for H3. If the test runs H3 with 64k flow control window, it will be blocked at ~32kB because of CC. As QUICHE needs to ACK packets by itself, not running the event loop of client connection in this test will cause the connection stuck. That's why I uses a smaller flow control window for H3.

To rule out the impact of CC window, I alter the test to still use 64k flow control window for H2 but use 32k for Quic.